actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -54,7 +54,7 @@ def parameters
                  rescue EOFError
                    query_parameters.dup
                  end
-        params.merge!(path_parameters)
+        params.merge!(set_utf8_encoding(path_parameters))
```

Comment:
`path_parameters` is already instance method, why are you sending it as argument to `set_utf8_encoding` method which is another instance method?